Firestore/Example/Podfile

```diff
@@ -29,17 +29,50 @@ post_install do |installer|
   end
 end
 
+# Returns true if the user has explicitly requested local sources or if this is
+# a non-PR Travis build.
+def use_local_sources()
+  return ENV.has_key?('USE_LOCAL_SOURCES') || ENV['TRAVIS_PULL_REQUEST'] == 'false'
```

Comment:
Looks like this is not working for cron builds

Reply:
In this [cron build](https://travis-ci.org/firebase/firebase-ios-sdk/jobs/576121850) in the ` before_install` output I see it working:

```
Fetching podspec for `FirebaseAuth` from `../..`
Fetching podspec for `FirebaseAuthInterop` from `../..`
Fetching podspec for `FirebaseCore` from `../..`
Fetching podspec for `FirebaseCoreDiagnostics` from `../..`
Fetching podspec for `FirebaseCoreDiagnosticsInterop` from `../..`
Fetching podspec for `FirebaseFirestore` from `../../`
Fetching podspec for `FirebaseFirestoreSwift` from `../../`
Fetching podspec for `GoogleBenchmark` from `GoogleBenchmark.podspec`
Fetching podspec for `GoogleDataTransport` from `../..`
Fetching podspec for `GoogleDataTransportCCTSupport` from `../..`
Fetching podspec for `GoogleTest` from `GoogleTest.podspec`
Fetching podspec for `GoogleUtilities` from `../..`
Fetching podspec for `LibFuzzer` from `LibFuzzer.podspec`
Fetching podspec for `ProtobufCpp` from `ProtobufCpp.podspec`
```

Could you point me at an instance where it's not working?

Reply:
https://travis-ci.org/firebase/firebase-ios-sdk/jobs/576121863

Reply:
Oh, I thought you meant the mechanism for detecting non-PR builds wasn't working, ha.

Yeah, so the problem here is that I'm editing the dependency out of the podspec but not here. I'll send a fix.

Reply:
Sounds good! And on closer inspection of the logs, the Xcode 8 build starting failing on all post-submits after this merge - not just the cron.